packages/compass/src/main/menu.ts

```diff
@@ -395,6 +395,14 @@ function viewSubMenu(
         },
       },
       separator(),
+      {
+        label: '&Toggle Sidebar',
```

Comment:
Something I noticed is that there's nothing in the menu that currently gets hidden depending on context. ie. disconnect is always visible even when you're not connected. And now Toggle Sidebar is also always visible even when you're not connected and that's kinda non-sensical when you you're looking at the connections sidebar.

Reply:
OK so actually there is some precedent: https://github.com/mongodb-js/compass/blob/0b6f35b72781a0930cccf5767e328eb631e45bb6/packages/compass/src/main/menu.ts#L533

called here https://github.com/mongodb-js/compass/blob/0b6f35b72781a0930cccf5767e328eb631e45bb6/packages/compass-home/src/components/home.tsx#L92